mods/ra/rules/aircraft.yaml

```diff
@@ -47,11 +48,11 @@ BADR.Bomber:
 	Aircraft:
 		CruiseAltitude: 2560
 		TurnSpeed: 5
-		Speed: 149
+		Speed: 180
```

Comment:
This makes it very hard to shoot down badgers before they have dropped their payload. Did you/the people who tested the changes (have there been any tests?) consider to nerf their health as a counter measure?

Reply:
Shooting down badgers to prevent damage is only really a factor in team games. Now that you can choose the angle of bombing that might not matter anymore either.

It might be more worthwhile to talk about making badgers untargetable like spy planes.

Reply:
This would allow a team of Ukrainian players to coordinate and destroy most buildings without leaving their opponents any chance to defend. I would agree that parabombs are broken atm without a minimal approach distance. The speed boost makes them even stronger (they ignore 4 AA guns) but is probably not game breaking, so go ahead.

Reply:
Right. I guess my argument is that being able to choose the direction will cause that anyway, even without the speed boost.

Something to keep an eye on.

Reply:
Current parabombs die to 3 aa guns if they have full line of sight and they're flying on top of them. With the speed increase one badger lives but is left almost dead. 4 aa guns still obliterate them.

Reply:
Whether they need a health decrease depends much more on the fact that players are getting the ability to control their direction rather then on the speed increase. For this cycle I suggest leaving health as is and observe the consequences.

Reply:
I didn't test with 4 AA guns next to each other - I should have posted the release vs patch gifs. I agree with Orb's last comment so don't think it matters anyway.

Reply:
> Whether they need a health decrease depends much more on the fact that players are getting the ability to control their direction rather then on the speed increase. For this cycle I suggest leaving health as is and observe the consequences.

I noted health specifically in the context of tge speed boost since it has a immediate effect on how much damage they take. That directional parabombs without mininal distance will break maps like Oregardens is out of question and not a problem of this PR.

Reply:
> I would agree that parabombs are broken atm without a minimal approach distance.

Minimal approach distance doesn't solve the 

> This would allow a team of Ukrainian players to coordinate and destroy most buildings without leaving their opponents any chance to defend

Reply:
It would not solve all the potential issues with the directional parabombs that I became a bit worried about (the directional cursor feature is great, don't get me wrong!). A minimal approach would however establish a "safe zone" where you could place your tech structures. Badgers would need to enter the zone from a angle where you can better defend against them.

Reply:
@matjaeck Do note that increasing the speed means there is a bigger distance between the bombs as well and a single position takes less damage ultimately.

Reply:
@GraionDilach this PR (at least as far as I know) counters it by reducing reload delay for parabombs.